### PR TITLE
feat(ui): update oss login page

### PR DESCRIPTION
**Pull request by alexpaxton**
Completes half the work described in #17151 
The other half is contingent upon this issue: https://github.com/influxdata/clockface/issues/455

- Use `FunnelPage` component in the OSS login page
- Use `InfluxDBLogo` from Clockface in login panel
- Factor out `SplashPage` components and styles

Screen Shot 2020-03-16 at 12 06 40 PM

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/onboarding/components/SigninForm.scss [hunk 1]
```diff
@@ -1,8 +1,38 @@
-.cf-panel.signin-panel {
+.signin-page {
+  .cf-funnel-page--content {
+    display: flex;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .cf-dapper-scrollbars--content {
+    display: flex !important;
```
Comment (hoorayimhelping) [inline ui/src/onboarding/components/SigninForm.scss]: is  this `!important` necessary?

File: ui/src/onboarding/components/SigninForm.scss [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+
+.signin-page--cubo {
+  background-image: url('../../assets/images/auth-logo.svg');
+  background-size: 100% 100%;
+  background-position: center center;
+  background-repeat: no-repeat;
+  width: 100px;
+  height: 100px;
+  display: inline-block;
+}
```
Comment (hoorayimhelping) [inline ui/src/onboarding/components/SigninForm.scss]: ninor nit: wanna alphabetize these selectors?

File: ui/src/onboarding/containers/SigninPage.tsx [hunk 3]
```diff
@@ -67,18 +68,18 @@ class SigninPage extends PureComponent<Props, State> {
...
       >
-        <SplashPage>
-          <Panel className="signin-panel">
+        <FunnelPage className="signin-page">
+          <Panel className="signin-page--panel">
             <Panel.Body alignItems={AlignItems.Center}>
-              <SplashPage.Logo />
-              <SplashPage.Header title="InfluxData" />
+              <div className="signin-page--cubo" />
+              <InfluxDBCloudLogo cloud={false} className="signin-page--logo" />
```
Comment (hoorayimhelping) [inline ui/src/onboarding/containers/SigninPage.tsx]: what is `cloud={false}` we have a `CLOUD` global variable that denotes whether we're in a cloud context or not.

Comment (alexpaxton) [reply ui/src/onboarding/containers/SigninPage.tsx]: that prop controls whether the word "Cloud" renders in the image. https://influxdata.github.io/clockface/?path=/story/graphics-brand-logos--influxdb-cloud

In this instance I left it `false` cuz this page is OSS only

File: ui/src/onboarding/components/SigninForm.scss (hunk 2 above)
Comment (alexpaxton) [reply ui/src/onboarding/components/SigninForm.scss]: I can do that, though makes me think we could benefit from more thorough CSS formatting. I think prettier supports that? Maybe a nice little tooling improvement

File: ui/src/onboarding/components/SigninForm.scss (hunk 1 above)
Comment (alexpaxton) [reply ui/src/onboarding/components/SigninForm.scss]: Unfortunately yes, in clockface there's an `!important` present to override styles from a library, so the madness trickles down

**Comment by alexpaxton** [comment]
@hoorayimhelping just pushed a commit that sorts the css attributes alphabetically

File: ui/src/onboarding/components/SigninForm.scss (hunk 2 above)
Comment (hoorayimhelping) [reply ui/src/onboarding/components/SigninForm.scss]: @alexpaxton deffo. I usually just say "alphabetize absent any other organizing scheme. would be rad to have one in place"

**Pull request merged**